lib/commands/exam.js

```diff
@@ -35,11 +37,17 @@ module.exports = TestCommand.extend({
   availableOptions: [
     { name: 'split',        type: Number,                  description: 'A number of files to split your tests across.' },
     { name: 'partition',    type: [Array, Number],         description: 'The number of the partition(s) to run after splitting.' },
-    { name: 'weighted',     type: Boolean,                 description: 'Weights the type of tests to help equal splits.' },
     { name: 'parallel',     type: Boolean, default: false, description: 'Runs your split tests on parallel child processes.' },
+    { name: 'load-balance', type: Number,                  description: 'Loads your test balanced' },
```

Comment:
That description isn't very informative. How about `Uses weighted load balancing to load tests in weighted order, slowest (acceptance) to fastest (lint).`